ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-commands.sh

```diff
@@ -16,19 +16,14 @@ BASE_DOMAIN=$(<"${SHARED_DIR}"/basedomain.txt)
 
 source "${SHARED_DIR}/nutanix_context.sh"
 
-RHCOS_PATCH=""
-if [[ ! -z "${OVERRIDE_RHCOS_IMAGE}" ]]; then
```

Comment:
The `OVERRIDE_RHCOS_IMAGE` is from `nutanix_context.sh`'s `override_rhcos_image`, but where `override_rhcos_image` is from?

Reply:
And actually if `OVERRIDE_RHCOS_IMAGE` is empty, would not affect the install.

Reply:
It's from vault, but seems like vault can not access today.

Reply:
Maybe I can remove it from vault

Reply:
Got it, it was gotten from `NUTANIX_AUTH_PATH=${CLUSTER_PROFILE_DIR}/secrets.sh`.

Agree, we should remove it from vault, actually from the beginning, the design of putting `override_rhcos_image` in some vault key sounds not good enough, because the `ipi-conf-nutanix-context` is being used in different versions of testing, while vault is shared with all versions. Even want `override_rhcos_image` supported, should be introduced as the step's environment ENV.